src/ServicePulse.Host/app/js/services/service.toast.js

```diff
@@ -32,6 +32,14 @@
     ];
 
     angular.module('sc')
-        .service('toastService', service);
+        .service('toastService', service)
```

Comment:
tbh I don't see value in toastService right now. That would be an easy dependency to get rid of.